### PR TITLE
dev-python/prettytable: version bump to 2.0.0, add pypy3 PYTHON_COMPAT

### PR DESCRIPTION
**Pull request by wimmuskee**
Signed-off-by: Wim Muskee <wimmuskee@gmail.com>

### CODE REVIEW

**Comment by wimmuskee** [comment]
updated for 2.0.0 bump (released yesterday) instead of 1.0.1

File: dev-python/prettytable/prettytable-2.0.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm64 ~hppa ~ia64 ~ppc64 ~sparc ~x86 ~amd64-linux ~x86-linux"
+
+RDEPEND="
+	dev-python/wcwidth[${PYTHON_USEDEP}]
+"
+BDEPEND="
+	dev-python/setuptools_scm[${PYTHON_USEDEP}]
+	test? ( ${RDEPEND} )
```
Comment (mgorny) [inline dev-python/prettytable/prettytable-2.0.0.ebuild:24]: This is redundant, d_e_t takes care of it.

**Review by mgorny — CHANGES_REQUESTED** [review]
```
===================================================================== ERRORS ======================================================================
___________________________________________________ ERROR collecting tests/test_prettytable.py ____________________________________________________
tests/test_prettytable.py:11: in <module>
    from prettytable import (
../prettytable-2.0.0-pypy3/lib/prettytable/__init__.py:43: in <module>
    __version__ = pkg_resources.get_distribution(__name__).version
/usr/lib/pypy3.7/site-packages/pkg_resources/__init__.py:465: in get_distribution
    dist = get_provider(dist)
/usr/lib/pypy3.7/site-packages/pkg_resources/__init__.py:341: in get_provider
    return working_set.find(moduleOrReq) or require(str(moduleOrReq))[0]
/usr/lib/pypy3.7/site-packages/pkg_resources/__init__.py:884: in require
    needed = self.resolve(parse_requirements(requirements))
/usr/lib/pypy3.7/site-packages/pkg_resources/__init__.py:770: in resolve
    raise DistributionNotFound(req, requirers)
E   pkg_resources.DistributionNotFound: The 'prettytable' distribution was not found and is required by the application
============================================================= short test summary info =============================================================
ERROR tests/test_prettytable.py - pkg_resources.DistributionNotFound: The 'prettytable' distribution was not found and is required by the applic...
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
================================================================ 1 error in 2.84s =================================================================
```

Sounds like it needs install-for-testing love.

File: dev-python/prettytable/prettytable-2.0.0.ebuild (hunk 1 above)
Comment (wimmuskee) [reply dev-python/prettytable/prettytable-2.0.0.ebuild]: removed it

**Comment by wimmuskee** [comment]
@mgorny  added the distutils_install_for_testing in the python_test phase, also running the tests require sqlite support to be available in the interpreter

**Review by mgorny — APPROVED** [review]
Cool work, thanks!

**Pull request closed without merging**